examples/apache-directory-list/webdav.conf

```diff
@@ -13,4 +13,15 @@
         Options +Indexes
```

Comment:
See comments for `-local`.

Reply:
? sorry I can find what you mean could you give me a link

Reply:
I just meant, all the comments for `apache-directory-list-local` are also relevant here!

Reply:
Yes correct :)